org-formation/300-account-defaults/_tasks.yaml

```diff
@@ -22,7 +22,7 @@ Ec2NoVpcDefaults:
 
 EbsVolumeCleanup:
   Type: update-stacks
-  Template: !Sub 'https://${AdminCentralCfnBucket}.s3.amazonaws.com/lambda-ebs-cleanup/0.0.1/lambda-ebs-cleanup.yaml'
+  Template: !Sub 'https://${AdminCentralCfnBucket}.s3.amazonaws.com/lambda-ebs-cleanup/0.0.2/lambda-ebs-cleanup.yaml'
```

Comment:
up to you, but you can just set to `master` until you get all the issues ironed out.  once it's working make a final tag and pin it to the tag.